.github/workflows/ci.yml

```diff
@@ -17,12 +17,14 @@ jobs:
       fail-fast: false
       matrix:
         ocaml-version: [4.11.1, 4.12.0]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
Nice, note that due to OCaml particularities is likely that in the future we will require pattern matching, so we will support only 3.10, IMHO that'd be fine, but of course let's see what people think.

Reply:
I was just tossing stuff out to see what would work, and the `build_pycoq` step worked on `3.6` lmao.